rules_building_block/defense_evasion_masquerading_windows_dll.toml

```
@@ -40,9 +40,24 @@ library where event.action == "load" and dll.Ext.relative_file_creation_time <=
         "?:\\$WINDOWS.~BT\\Work\\*",
         "?:\\Windows\\WinSxS\\*",
         "?:\\Windows\\SoftwareDistribution\\Download\\*",
-        "?:\\Windows\\assembly\\NativeImages_v*"
+        "?:\\Windows\\assembly\\NativeImages_v*", 
+        "E:\\docker\\windowsfilter\\*", 
+        "E:\\System\\Users\\*",
+        "\\\\?\\Volume{*}\\Windows\\*"
```

```suggestion
        "?:\\Windows\\assembly\\NativeImages_v*"
```
Those seem to be env-specific, each of them only occurs in one cluster

---

`\\\\?\\Volume{*}\\Windows\\` is not env specific, we can limit to `C:\` drive too but we will miss executed from mounted ISO/VHD files.